treeherder/perf/models.py

```diff
@@ -213,10 +213,18 @@ class PerformanceAlertSummary(models.Model):
                 (FIXED, 'Fixed'),
                 (BACKED_OUT, 'Backed out'))
 
+    BUGZILLA = 0
+    GITHUB = 1
+
+    ISSUE_TRACKERS = ((BUGZILLA, 'Bugzilla'),
+                      (GITHUB, 'Github'))
```

Comment:
I think it's probably worth doing a bit more work upfront to parameterize this by creating an actual model for the issue tracker, and then referencing it from here. That will make it easier to support more than one github repository in the future. Something like this in `model/models.py`:

```py
class IssueTracker(models.Model):
    name = models.CharField(...)  # e.g. `bugzilla.mozilla.org`, `servo`
    base_url = models.URLField()  # e.g. `https://bugzilla.mozilla.org/show_bug.cgi?id=`
```

You could then reference it via a foreign key field in this model:

```py
class PerformanceAlertSummary(models.Model):
   ...
   issue_tracker = models.ForeignKeyField(IssueTracker)
```

You can create the initial set of values you want (servo, bugzilla) in the fixtures here:

https://github.com/mozilla/treeherder/tree/master/treeherder/model/fixtures

I think it's ok to leave the frontend code more or less as-is for now (i.e. hardcoded) as it's much easier to change.